src/chainparams.cpp

```diff
@@ -457,6 +457,13 @@ class CDevNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nWindowSize = 100;
         consensus.vDeployments[Consensus::DEPLOYMENT_BIP147].nThreshold = 50; // 50% of 100
 
+        // Deployment of DIP0003
+        consensus.vDeployments[Consensus::DEPLOYMENT_DIP0003].bit = 3;
+        consensus.vDeployments[Consensus::DEPLOYMENT_DIP0003].nStartTime = 1518649200; // Feb 15th, 2018
+        consensus.vDeployments[Consensus::DEPLOYMENT_DIP0003].nTimeout = 1897340400; // Feb 15th, 2030
```

Comment:
Because devnets get created/destroyed all the time and thus the deployment starts from scratch for each one. Should actually be like 100 years or so...Or 1000? :)

What do you mean with bip9 requires +1 year? Is this in-code or "only" defined in the BIP?

Reply:
Not sure about the code, but in the BIP this is done so that deployment bits can be re-used in the future after they have been deployed and timeout.

Reply:
Thinking more about this...we'll eventually ditch the whole deployment logic after the deployment is activated, spork15 got activated and we do the general cleanup (removing all the backwards compatibility code). At that point, this bit will also get free again.

Reply:
Ok, will change it to one year. But as said, best is if we can ditch the whole BIP9 logic and replace it with a block height after 12.4 is fully deployed.